apps/typescriptcourse/src/pages/tutorials/[module]/[lesson]/index.tsx

```diff
@@ -14,7 +16,12 @@ export const getStaticProps: GetStaticProps = async (context) => {
   const lesson = await getLesson(lessonSlug)
 
   return {
-    props: {lesson, module},
+    props: {
+      lesson,
+      module,
+      videoResourceId: lesson.videoResourceId,
+      transcript: lesson.transcript,
```

Comment:
we want this data at build time, but we use a more general type in the presentation layer (`LessonResource`) so these specific bits of media aren't on the type so not available